tests/rptest/tests/partition_move_interruption_test.py

```diff
@@ -65,26 +65,32 @@ def _throttle_recovery(self, new_value):
     @cluster(num_nodes=7, log_allow_list=RESTART_LOG_ALLOW_LIST)
     @matrix(replication_factor=[1, 3],
             unclean_abort=[True, False],
-            recovery=[NO_RECOVERY, RESTART_RECOVERY])
+            recovery=[NO_RECOVERY, RESTART_RECOVERY],
+            compacted=[False, True])
     def test_cancelling_partition_move(self, replication_factor, unclean_abort,
-                                       recovery):
+                                       recovery, compacted):
         """
         Cancel partition moving with active consumer / producer
         """
         self.start_redpanda(num_nodes=5,
                             extra_rp_conf={
                                 "default_topic_replications": 3,
+                                "compacted_log_segment_size": 1 * (2 ^ 20)
```

Comment:
nit: could you add a comment why this value? Low to encourage compactions more quickly?

Also may be more readable as 1024 * 1024